src/content-tags-drawer/ContentTagsDrawer.tsx

```diff
@@ -70,10 +73,149 @@ const TaxonomyList = ({ contentId }) => {
+    >
+      <div className="d-flex justify-content-end">
+        { commitGlobalStagedTagsStatus !== 'loading' ? (
+          <Stack direction="horizontal" gap={2}>
+            <Button
+              className="font-weight-bold tags-drawer-cancel-button"
+              variant="tertiary"
+              onClick={isEditMode
+                ? toReadMode
+                : onClose}
+            >
+              { intl.formatMessage(isEditMode
+                ? messages.tagsDrawerCancelButtonText
+                : messages.tagsDrawerCloseButtonText)}
+            </Button>
+            <Button
+              variant="dark"
+              className="rounded-0"
+              onClick={isEditMode
+                ? commitGlobalStagedTags
+                : toEditMode}
+            >
+              { intl.formatMessage(isEditMode
+                ? messages.tagsDrawerSaveButtonText
+                : messages.tagsDrawerEditTagsButtonText)}
+            </Button>
+          </Stack>
+        )
+          : (
+            <Spinner
+              animation="border"
+              size="xl"
+              screenReaderText={intl.formatMessage(messages.loadingMessage)}
+            />
+          )}
+      </div>
+    </Container>
+  );
+};
+
+const ContentTagsComponentVariantFooter = () => {
+  const intl = useIntl();
+  const {
+    commitGlobalStagedTagsStatus,
+    commitGlobalStagedTags,
+    isEditMode,
+    toReadMode,
+    toEditMode,
+  } = useContext(ContentTagsDrawerContext);
+
+  return (
+    <div>
+      {isEditMode ? (
+        <div>
+          { commitGlobalStagedTagsStatus !== 'loading' ? (
+            <Stack direction="horizontal" gap={2}>
+              <Button
+                className="font-weight-bold tags-drawer-cancel-button"
+                variant="tertiary"
+                onClick={toReadMode}
+              >
+                {intl.formatMessage(messages.tagsDrawerCancelButtonText)}
+              </Button>
+              <Button
+                variant="dark"
+                className="rounded-0"
+                onClick={commitGlobalStagedTags}
+                block
+              >
+                {intl.formatMessage(messages.tagsDrawerSaveButtonText)}
+              </Button>
+            </Stack>
+          ) : (
+            <div className="d-flex justify-content-center">
+              <Spinner
+                animation="border"
+                size="xl"
+                screenReaderText={intl.formatMessage(messages.loadingMessage)}
+              />
+            </div>
+          )}
+        </div>
+      ) : (
+        <Button
+          variant="outline-primary"
+          onClick={toEditMode}
+          block
+        >
+          {intl.formatMessage(messages.manageTagsButton)}
+        </Button>
+      )}
+    </div>
+  );
 };
 
+interface ContentTagsDrawerProps {
+  id?: string;
+  onClose?: () => void;
+  variant?: 'drawer' | 'component';
```

Comment:
Nit:
```suggestion
  variant?: 'drawer' | 'sidebar';
```

We will probably use this on the Collections sidebar

Reply:
Let's leave it as a 'component'. This means that ContentTags will be used as a drawer (with close buttons, title, etc.) or as a component. It doesn't mean that it will be used for components, it can also be used in collections but as a component.